### DISCUSSION TITLE
电脑上安装了cuda并且能使用，但是ocr选择cuda时会报错,文本检测用cuda不会报错。软件提示modules中的ocr_mit.py中的 self.mode不存在。

### DISCUSSION THREAD

**Discussion by jiangnn1** [dmMaze/BallonsTranslator#526] [Q&A] [answered]
![微信图片_20240731111111](https://github.com/user-attachments/assets/2e442e8b-2dde-4174-a79a-9e98a3cb75f4)
![微信图片_20240731111259](https://github.com/user-attachments/assets/6f3aa0b0-1b2b-4a86-bb02-7f9e30708edb)

**Comment by dmMaze** [comment]
我可以正常跑啊，你用的是最新版吗，commit hash 是啥

**Reply by jiangnn1** [reply]
我是昨天下载的，commit hash 怎么看

**Reply by dmMaze** [reply, marked as answer]
> 我是昨天下载的，commit hash 怎么看

把 log 截全发上来

**Reply by jiangnn1** [reply]
我看了下我下载的时dev版本，是不是太老了

**Reply by jiangnn1** [reply]
![微信图片_20240731134733](https://github.com/user-attachments/assets/f989d0b7-d278-4a5a-af3c-8326c9c7c93d)

**Reply by dmMaze** [reply]
你是什么时候从下的这个包

**Reply by jiangnn1** [reply]
昨天下载的

**Reply by dmMaze** [reply]
> 昨天下载的

试试在标题栏运行，去掉翻译和OCR的勾跑一遍，看抹除文字结果正不正常

**Reply by dmMaze** [reply]
另外你是不是改代码了？

**Reply by jiangnn1** [reply]
> > 昨天下载的
> 
> 试试在标题栏运行，去掉翻译和OCR的勾跑一遍，看抹除文字结果正不正常

我选择cuda会立刻弹出这个错误，然后软件自动关闭了

**Reply by jiangnn1** [reply]
> 另外你是不是改代码了？

是的，一开始运行qt报错QFontmetricsF object has no attribute width，我找到报错位置将width改成了maxwidth

**Reply by jiangnn1** [reply]
> > > 昨天下载的
> > 
> > 
> > 试试在标题栏运行，去掉翻译和OCR的勾跑一遍，看抹除文字结果正不正常
> 
> 我选择cuda会立刻弹出这个错误，然后软件自动关闭了

全程使用cpu是没问题的

**Reply by dmMaze** [reply]
> 是的，一开始运行qt报错QFontmetricsF object has no attribute width，我找到报错位置将width改成了maxwidth

这是哪里的代码报的错？


> 我选择cuda会立刻弹出这个错误，然后软件自动关闭了

你的 torch 版本是？试试网盘里那个大包

**Reply by jiangnn1** [reply]
> > 是的，一开始运行qt报错QFontmetricsF object has no attribute width，我找到报错位置将width改成了maxwidth
> 
> 这是哪里的代码报的错？
> 我是昨天改的忘记在哪了
> > 我选择cuda会立刻弹出这个错误，然后软件自动关闭了
> 
> 你的 torch 版本是？试试网盘里那个大包
torch                         2.0.0+cu118
torchaudio                    2.0.1+cu118
torchvision                   0.15.0+cu118
网盘链接可以发下吗

**Reply by dmMaze** [reply]
网盘链接在 readme 里，先看看安装说明

**Reply by jiangnn1** [reply]
> 网盘链接在 readme 里，先看看安装说明

用网盘链接里面的可以用了，谢谢
